addons/account/models/account_move_line.py

```diff
@@ -1631,10 +1624,6 @@ def write(self, vals):
             if line.parent_state == 'posted' and any(self.env['account.move']._field_will_change(line, vals, field_name) for field_name in protected_fields['tax']):
                 line._check_tax_lock_date()
 
-            # Check the reconciliation.
-            if any(self.env['account.move']._field_will_change(line, vals, field_name) for field_name in protected_fields['reconciliation']):
```

Comment:
**if** we really want this, then we don't need to defined the `reconciliation` key from `protected_fields` anymore